odoo/addons/base/models/ir_model.py

```diff
@@ -298,7 +298,7 @@ def _add_manual_models(self):
 class IrModelFields(models.Model):
     _name = 'ir.model.fields'
     _description = "Fields"
-    _order = "name"
+    _order = "store desc, name"
```

Comment:
This is in order to avoid see the fields in full alphabetic order, then show first show the non-computed ones and then then the computed ones, that's useful to debug the more basic situations. You can ignore that if you are not agree.

Reply:
I understand your point but I am afraid it is confusing when you have a model like res.partner with the fields on multiple pages

Reply:
You got a point! I will revert that.

Reply:
You got a point, I will make the change.